# Do not put custom makefiles flags to empty by default (so EB can overwrite them)
# wpoely86@gmail.com
diff -ur lua-5.2.4.orig/src/Makefile lua-5.2.4/src/Makefile
--- lua-5.2.4.orig/src/Makefile	2013-11-11 12:45:49.000000000 +0100
+++ lua-5.2.4/src/Makefile	2019-04-09 10:04:57.701119382 +0200
@@ -19,9 +19,9 @@
 SYSLDFLAGS=
 SYSLIBS=
 
-MYCFLAGS=
-MYLDFLAGS=
-MYLIBS=
+MYCFLAGS?=
+MYLDFLAGS?=
+MYLIBS?=
 MYOBJS=
 
 # == END OF USER SETTINGS -- NO NEED TO CHANGE ANYTHING BELOW THIS LINE =======
